packages/graphqlsp/src/diagnostics.ts

```diff
@@ -40,6 +42,7 @@ export const SEMANTIC_DIAGNOSTIC_CODE = 52001;
 export const MISSING_OPERATION_NAME_CODE = 52002;
 export const MISSING_FRAGMENT_CODE = 52003;
 export const USING_DEPRECATED_FIELD_CODE = 52004;
+export const UNUSED_FIELD_CODE = 52005;
```

Comment:
question: why do these start at 52k? (just curious)

Reply:
No particular reason, hoping for no clashes 😅